src/sentry/issues/issue_occurrence.py

```diff
@@ -36,6 +38,9 @@ class IssueOccurrenceData(TypedDict):
     level: str | None
     culprit: str | None
     initial_issue_priority: NotRequired[int | None]
+    # Who to assign the issue to when creating a new issue. Has no effect on existing issues.
+    # In the format of an Actor identifier, as defined in `ActorTuple.from_actor_identifier`
+    assignee: NotRequired[str | None]
```

Comment:
Nit, i've written these as docstrings, since some IDE integrations do pick this up

```suggestion
    assignee: NotRequired[str | None]
    """
    Who to assign the issue to when creating a new issue. Has no effect on existing issues.
    In the format of an Actor identifier, as defined in `ActorTuple.from_actor_identifier`
    """
```

Reply:
Lol I hate this but I guess it's the standard the community decided on